BUILD.bazel

```diff
@@ -19,7 +19,7 @@ copy_file(
     out = "src/pcre2_chartables.c",
 )
 
-# Removed src/pcre2_ucptables.c below because it is #included in 
+# Removed src/pcre2_ucptables.c below because it is #included in
```

Comment:
Ensured that Detrail is run on all files that are part of the output tarball (...except the files where we deliberately don't want to run it, eg CRLF files, and any/all tool outputs from autoconf).

Reply:
Ideally I would like to see that ALL our files get `detrail`, which necessary implies we need to teach it to work also in Windows style line endings.

I can see there might be a few exceptions (ex: test files that require explicit trailing blank characters), but even those could be exempted at the line number level IMHO.

Reply:
We just have two nasty old `.bat` files in the repo with CRLF. They're OK for now (I have a task to convert them to `.ps1` files someday - which is vastly better than bat). The test files are treated as "binary" and so I haven't proactively detrailed them. But, all the other files in the repo have now been added to Detrail.